nvme.c

```diff
@@ -441,6 +447,8 @@ static int get_log(int argc, char **argv, struct command *cmd, struct plugin *pl
 		{"log-len",      'l', "NUM", CFG_POSITIVE, &cfg.log_len,      required_argument, log_len},
 		{"aen",          'a', "NUM", CFG_POSITIVE, &cfg.aen,          required_argument, aen},
 		{"raw-binary",   'b', "",    CFG_NONE,     &cfg.raw_binary,   no_argument,       raw_binary},
+		{"lsp",          's', "NUM", CFG_POSITIVE, &cfg.lsp,          required_argument, lsp},
+		{"lsp",          'o', "NUM", CFG_POSITIVE, &cfg.lpo,          required_argument, lpo},
```

Comment:
This is a u8, but CFG_POSITIVE will think it's pointing to a u32. It happens to be safe in this case because of struct padding, but it should be CFG_BYTE.

Reply:
Oh, and I think you intended the long option should be "lpo".